components/brave_wallet_ui/components/shared/nft-icon/nft-icon.tsx

```diff
@@ -16,7 +16,7 @@ import {
 
 // hooks
 import {
-  useGetIpfsGatewayTranslatedNftUrlQuery //
+  useGetIpfsGatewayTranslatedNftUrlQuery
```

Comment:
please leave the `//` as it prevents `prettier` from formatting this into one long line that is over 80 character long